data/human/syndicate outfits.txt

```diff
@@ -212,5 +212,5 @@ outfit "50KZ Electrostatic Steering"
 	"steering flare sprite" "effect/ion flare/huge"
 		"frame rate" 0.8
 	"steering flare sound" "ion huge"
-	description "This is the most powerful Electrostatic engine in human space, allowing almost any ship to move as if it were a fighter. It does, however, draw a small reactor's worth of energy all by itself."
-	description "	Electrostatic engines provide a kick between Plasma and Atomic engines, but require both maintenance energy and a considerable amount of energy when moving."
+	description "This is the most powerful electrostatic steering engine in human space, allowing almost any ship to turn as if it were a fighter. It does, however, draw a small reactor's worth of energy all by itself."
+	description "	Electrostatic engines are placed between Plasma and Atomic engines, but they require a considerable amount of energy to use, and they constantly draw power even when the ship is idle."
```

Comment:
I'm being stubborn; I want "provide a kick" instead of "are placed". The latter just comes off as very flat and plain to me.